network_manager/__init__.py

```diff
@@ -10,19 +10,18 @@ def main(global_config, **settings):
     Base.metadata.bind = engine
 
     config = Configurator(settings=settings)
+
     config.include("pyramid_chameleon")
 
-    # Add a route / path called node_all at /node/update
-    # This is the 'admin' page
-    # This is for displaying all entries in the database at the top of the page and a blank form
-    config.add_route("node_all", "/node/update")
+    config.add_route("node_edit", "/node/edit", request_method=("POST", "GET"))
+
+    config.add_route("node_info", "/node/{node_id}", request_method="GET")
```

Comment:
This route is not required by https://github.com/DACCS-Climate/roadmap/issues/22 but it's a nice extra so let's leave it in. I've separated it into a different route from /node though so that the user can easily get the information about *this* node (required by https://github.com/DACCS-Climate/roadmap/issues/22)